doc/generate_logos.py

```diff
@@ -15,11 +15,15 @@
 import logging
 import subprocess
 import sys
-
-
-default_source_dir = os.path.join(os.path.dirname(__file__), "src/logo")
+from platform import system
+
+if system()[0:3].lower() == "win":
+    default_source_dir = os.path.join(os.path.dirname(__file__), "src\logo")
+    default_output_dir = os.path.join(os.path.dirname(__file__), "_build\logo")
```

Comment:
You don't need this if statement. The whole point of `join` is to add the system specific path separator, so you can just use `os.path.join(os.path.dirname(__file__), "src", "logo")`.